### PR TITLE
Fantem MultiSensor ZB003-X

### PR DESCRIPTION
**Pull request by kirovilya**
https://github.com/Koenkk/zigbee-herdsman-converters/issues/2071

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -15202,6 +15202,37 @@ const devices = [
...
+            fz.ZB003X,
+            fz.ZB003X_attr,
+            fz.ZB003X_occupancy,
+        ],
+        toZigbee: [tz.ZB003X],
+        exposes: [
+            e.occupancy(), e.battery(),
+            e.illuminance().withUnit('lx'),
+            e.temperature(), e.humidity(),
+            exposes.numeric('reporting_time', exposes.access.ALL),
```
Comment (Koenkk) [inline devices.js:15225]: Can you also add description for all (use .withDescripition("...."))

File: devices.js [hunk 2]
```diff
@@ -15202,6 +15202,37 @@ const devices = [
...
+            exposes.numeric('temperature_calibration', exposes.access.ALL),
+            exposes.numeric('humidity_calibration', exposes.access.ALL),
+            exposes.numeric('illuminance_calibration', exposes.access.ALL),
+            exposes.binary('pir_enable', exposes.access.ALL, true, false),
+            exposes.binary('led_enable', exposes.access.ALL, true, false),
+            exposes.binary('reporting_enable', exposes.access.ALL, true, false),
+            exposes.enum('sensitivity', exposes.access.ALL, ['low', 'medium', 'high']),
+            exposes.enum('keep_time', exposes.access.ALL, ['0', '30', '60', '120', '240']),
```
Comment (Koenkk) [inline devices.js:15233]: `exposes.access.ALL` should be `exposes.access.STATE_SET` since there is no convertGet

File: devices.js (hunk 1 above)
Comment (kirovilya) [reply devices.js:15225]: ohh :)

File: devices.js [hunk 3]
```diff
@@ -15202,6 +15202,37 @@ const devices = [
...
+            fz.ZB003X,
+            fz.ZB003X_attr,
+            fz.ZB003X_occupancy,
+        ],
+        toZigbee: [tz.ZB003X],
+        exposes: [
+            e.occupancy(), e.battery(),
+            e.illuminance().withUnit('lx'),
+            e.temperature(), e.humidity(),
+            exposes.numeric('reporting_time', exposes.access.ALL).withDescripition("Reporting interval in minutes"),
```
Comment (Koenkk) [inline devices.js:15225]: Feel free to add `// eslint-disable-next-line` above the lines that are too long

File: converters/fromZigbee.js [hunk 4]
```diff
@@ -4949,6 +4949,79 @@ const converters = {
...
+    ZB003X: {
+        cluster: 'manuSpecificTuya',
+        type: ['raw'],
+        convert: (model, msg, publish, options, meta) => {
+            const dp = msg.data[5];
+            const value = tuya.getDataValue(tuya.dataTypes.value, msg.data.slice(8));
+            let val;
+            switch (dp) {
+            case 107: // 0x6b temperature
+                return {temperature: (value / 10).toFixed(1)};
```
Comment (Koenkk) [inline converters/fromZigbee.js:4961]: Please use `calibrateAndPrecisionRoundOptions(temperature, options, 'temperature')` here

File: converters/fromZigbee.js [hunk 5]
```diff
@@ -4949,6 +4949,79 @@ const converters = {
...
+            case 108: // 0x6c humidity
+                return {humidity: value};
```
Comment (Koenkk) [inline converters/fromZigbee.js:4963]: Please use `calibrateAndPrecisionRoundOptions(humidity, options, 'humidity')`

**Comment by Koenkk** [comment]
Thanks!

File: converters/fromZigbee.js [hunk 6]
```diff
@@ -4949,6 +4949,79 @@ const converters = {
...
+            }
+        },
+    },
+    ZB003X_occupancy: {
+        cluster: 'ssIasZone',
+        type: 'commandStatusChangeNotification',
+        convert: (model, msg, publish, options, meta) => {
+            const zoneStatus = msg.data.zonestatus;
+            return {
+                occupancy: (zoneStatus & 1<<2) > 0,
```
Comment (dsantos21) [inline converters/fromZigbee.js]: Hi! I've just updated the issue #2071 with some comments about occupancy not working. After some debug, I changed my local instance and now it's working (without the bitwase operator):

occupancy: (zoneStatus & 1) > 0,

**Pull request merged**